README.md

```diff
@@ -1 +1,3 @@
-# layout
+layout
+=========
```

Comment:
Add a blank line after the `======` as done in other element repos like `core-icon`: https://raw.githubusercontent.com/Polymer/core-icon/master/README.md